.github/workflows/qit.yml

```
@@ -54,7 +54,7 @@ jobs:
       - name: Set PHP version
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 7.4
+          php-version: 7.3
```

This needs to stay at 7.4, as that's the minimum required for QIT
```suggestion
          php-version: 7.4
```